R/utils.r

```diff
@@ -42,7 +42,51 @@ render_template <- function(name, data = list()) {
 }
 
 is_installed <- function(pkg, version = 0) {
-  system.file(package = pkg) != "" && packageVersion(pkg) > version
+  installed_version <- tryCatch(utils::packageVersion(pkg), error = function(e) NA)
+  !is.na(installed_version) && installed_version >= version
+}
+
+check_suggested <- function(pkg, version = NULL, compare = NA) {
+
+  if (is.null(version)) {
+    if (!is.na(compare)) {
+      stop("Cannot set ", sQuote(compare), " without setting ",
+           sQuote(version), call. = FALSE)
+    }
+
+    dep <- suggests_dep(pkg)
+
+    version <- dep$version
+    compare <- dep$compare
+  }
+
+  if (!check_dep_version(pkg, version, compare)) {
+    msg <- paste0(sQuote(pkg),
+      if (version == 0) "" else paste0(" >= ", version),
+      " must be installed for this functionality.")
+
+    if (interactive()) {
+      message(msg, "\nWould you like to install it?")
+      if (menu(c("Yes", "No")) == 1) {
+        install.packages(pkg)
+      }
```

Comment:
Yes! Fixed with https://github.com/jimhester/devtools/commit/ab8bae8508b9b656da095a2cb68d4163aaf74eb6. Having the two `stop()` calls right after each other looks a little ugly, but modifying the conditional to avoid them would be just as bad in other ways.